sdk/core/azure-core/azure/core/pipeline/policies/_redirect.py

```diff
@@ -30,22 +30,33 @@
 from urllib.parse import urlparse
 
 from azure.core.exceptions import TooManyRedirectsError
+from azure.core.pipeline import PipelineResponse, PipelineRequest
 
 from ._base import HTTPPolicy, RequestHistory
+from ._utils import get_domain
 
 
 _LOGGER = logging.getLogger(__name__)
 
 
+def domain_changed(original_domain: str, url: str) -> bool:
+    domain = get_domain(url)
```

Comment:
Please run next-pylint to be sure you're not introducing things that will break in two weeks. In particular, this one should have a docstring.